crates/yakui-core/src/paint/paint_dom.rs

```diff
@@ -203,6 +204,10 @@ impl PaintDom {
             let mut pos = vertex.position * self.scale_factor;
             pos += self.unscaled_viewport.pos();
             pos /= self.surface_size;
+            if mesh.pipeline == Pipeline::Text {
+                pos = (pos * self.surface_size).round() / self.surface_size;
```

Comment:
Since we divide by surface_size above, can we move this up statement a line and just write:
```rust
pos = pos.round();
```

Reply:
To be more specific, this code:

```rust
pos /= self.surface_size;
if mesh.pipeline == Pipeline::Text {
    pos = (pos * self.surface_size).round() / self.surface_size;
}
```

would behave identically to this code:

```rust
if mesh.pipeline == Pipeline::Text {
    pos = pos.round();
}
pos /= self.surface_size;
```